tools/generate_pv_yaml_rest.sh

```diff
@@ -0,0 +1,328 @@
+#!/bin/bash
+#
+# Copyright 2021 IBM Corp.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+usage() {
+  echo "Usage: $0
+                -f|--filesystem <Name of Volume's Source Filesystem>
+                -l|--linkpath <full Path of Volume in Primary Filesystem>
+                -F|--fileset <name of source fileset>
+                -s|--size <size in GB>
+                -u|--username <Username of spectrum scale GUI user account.>
+                -t|--password <Password of spectrum scale GUI user account.>
+                -r|--route <Route host name used to route traffic to the spectrum scale GUI service.>
+                [-p|--pvname <name for pv>]
+                [-c|--storageclass <StorageClass for pv>]
+                [-a|--accessmode <AccessMode for pv>]
+                [-h|--help] " 1>&2
+  exit 1
+}
+
+fullUsage() {
+  echo "Usage: $0
+                -f|--filesystem <Name of Volume's Source Filesystem>
+                -l|--linkpath <full Path of Volume in Primary Filesystem>
+                -F|--fileset <name of source fileset>
+                -s|--size <size in GB>
+                -u|--username <Username of spectrum scale GUI user account.>
+                -t|--password <Password of spectrum scale GUI user account.>
+                -r|--route <Route host name used to route traffic to the spectrum scale GUI service.>
+                [-p|--pvname <name for pv>]
+                [-c|--storageclass <StorageClass for pv>]
+                [-a|--accessmode <AccessMode for pv>]
+                [-h|--help]
+
+
+Example 1: Single Fileystem
+	In this setup there is only one fileystem 'gpfs0' and directory from the same fileystem is being used as volume.
+
+	$0 --filesystem gpfs0 --linkpath /ibm/gpfs0/fileset1/.volumes/staticpv --size 10 --pvname mystaticpv --username admin --password password --route ibm-spectrum-scale-gui-ibm-spectrum-scale.apps.hci-cluster.cp.fyre.ibm.com
```

Comment:
i think, this is not a good idea to take password in command line

Reply:
Password is accepted using --password flag, if flag not passed then code for prompt to enter password is added. Password is read securely without display on screen.

Reply:
```sh
bash generate_pv-yaml_rest.sh --filesystem vdisknsd-sample --linkpath /mnt/vdisknsd-sample/test --size 10 --pvname mystaticpv --guihost ibm-spectrum-scale-gui-ibm-spectrum-scale.apps.hci-cluster.cp.fyre.ibm.com
GUI Username: testadmin
GUI Password: 
FSETNAME=
INFO: volumeHandle: 17928868485131832816;0A110FA6:60FABED4;path=/mnt/vdisknsd-sample/test
INFO: Successfully created mystaticpv.yaml
INFO: Successfully created pvc-mystaticpv.yaml
```

Reply:
Thanks @nitishkumar4  for making the changes.